pkg/apis/config/v1alpha2/meshrootcertificate.go

```diff
@@ -122,6 +122,14 @@ type TresorCASpec struct {
 // can be (Active, Passive).
 type MeshRootCertificateIntent string
 
+const (
+	// Passive is the passive option for the Intent of the MeshRootCertificate
+	Passive MeshRootCertificateIntent = "passive"
```

Comment:
I'd prefer adding prefixes to these variables for readability. `v1alpha2.MRCIntentPassive` is a clearer variable name than `v1alpha2.Passive`